zio-schema-examples/shared/src/main/scala/dev/zio/schema/example/example4/Example4Transformation.scala

```diff
@@ -65,13 +65,13 @@ object Example4Transformation extends zio.App {
   val domainPerson: Either[String, DomainPerson] =
     WebPerson.schema.migrate(personTransformation).flatMap(f => f(webPerson))
 
-  override def run(args: List[String]): URIO[zio.ZEnv, ExitCode] = ZIO.debug(domainPerson).exitCode
+  override def run: URIO[Any, ExitCode] = ZIO.debug(domainPerson).exitCode
```

Comment:
Return type of `run` is actually `Any`, not `ExitCode`. No need to call `exitCode` anymore.